demo/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "demo",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "aphrodite": "^2.2.0",
+    "react": "^16.3.2",
+    "react-dom": "^16.3.2",
+    "react-scripts": "1.1.4",
+    "wonder-blocks-button": "../packages/wonder-blocks-button",
+    "wonder-blocks-core": "../packages/wonder-blocks-core"
```

Comment:
🤷‍♂️ It'd probably be good to import all of the packages that we provide, making them automatically available for all to use.

(Sidebar: I think we may want to have a script that auto-creates a new WB package and handles details like this.)

Reply:
I'm not sure how to make them automatically available.  I'm going to add all of them manually right now and will create a script for adding new packages in the future that updates this.

Reply:
Manually is fine for now!